components/threading/global_threads/thread_pane/thread_pane.test.tsx

```diff
@@ -85,6 +92,7 @@ describe('components/threading/global_threads/thread_pane', () => {
                     },
                 },
                 users: {
+                    profiles,
```

Comment:
would you be open in writing a unit test which return shouldFilterJoinLeavePost true or false?